internal/dockerfile/Dockerfile.tmpl

```diff
@@ -26,6 +26,38 @@ RUN {{ if .UseBuildKit }}--mount=type=cache,target=/go/pkg/mod \
 {{ end -}}
 ################################################################################
 
+# To only build the tests run: docker build . --target test
+FROM {{ .DockerHubMirror }}golang:{{ .Constants.DefaultGoVersion }}-alpine{{ .Constants.DefaultAlpineImage }} AS test
```

Comment:
Sandro told me in person that we are not doing `FROM builder AS test` in order for the `make prepare-static-check` part to be cacheable during interactive use at development time. This should be noted in a code comment for future reference.